src/Microsoft.DotNet.Interactive.CSharpProject/Build/BuildResult.cs

```diff
@@ -148,22 +146,25 @@ public AdhocWorkspace CreateWorkspace()
         return workspace;
     }
 
-    private static ProjectInfo CreateProjectInfo(ProjectBuildInfo projectBuildInfo, CodeAnalysis.Workspace workspace, ProjectId projectId)
+    private static ProjectInfo CreateProjectInfo(
+        ProjectBuildInfo projectBuildInfo, 
+        Solution solution, 
+        ProjectId projectId)
     {
-        string projectName = Path.GetFileNameWithoutExtension(projectBuildInfo.ProjectFilePath);
+        var projectName = Path.GetFileNameWithoutExtension(projectBuildInfo.ProjectFilePath);
 
         return ProjectInfo.Create(
             projectId,
-            VersionStamp.Create(),
+            VersionStamp.Default,
             projectName,
             projectName,
             LanguageNames.CSharp,
             filePath: projectBuildInfo.ProjectFilePath,
             outputFilePath: projectBuildInfo.TargetPath,
             documents: GetDocuments(projectBuildInfo, projectId),
-            projectReferences: GetExistingProjectReferences(projectBuildInfo, workspace),
+            projectReferences: GetExistingProjectReferences(projectBuildInfo, solution),
             metadataReferences: GetMetadataReferences(projectBuildInfo),
-            analyzerReferences: GetAnalyzerReferences(projectBuildInfo, workspace),
+            analyzerReferences: null,
```

Comment:
Is this intentional? Are we ignoring analyzers because while analyzers can block build for projects that otherwise compile, the opposite is not true (i.e. analyzers can't cause projects that fail to build because of other build failures to build successfully)?

Reply:
Analyzers are not supported for security reasons.

Reply:
Analyzers are not supported for security reasons.